src/CollAction/Controllers/ProjectsController.cs

```diff
@@ -160,9 +184,15 @@ public ViewResult StartInfo()
         [HttpPost]
         [ValidateAntiForgeryToken]
         [Authorize]
-        public async Task<IActionResult> Edit(int id, [Bind("Id,Name,Description,Goal,CategoryId,LocationId,Target,End")] Project project)
+        public async Task<IActionResult> Edit(int id, [Bind("Id,OwnerId,Name,Description,Goal,CategoryId,LocationId,Target,End")] EditProjectViewModel editProjectViewModel)
```

Comment:
Here you're taking the owner id as parameter. This means the user could change the owner-id from the form, and circumvent the owner validation! Security vulnerability.